### PR TITLE
Follow-up #324: better version handling

### PR DESCRIPTION
**Pull request by MikhailGoncharov**
Follow-up for https://github.com/SAP-samples/cloud-cap-samples/pull/324

### CODE REVIEW

File: test/consuming-services.test.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ describe('Consuming Services locally', () => {
             })
         })
     }).where(`name like`, 'E%')
-    if (cds.version >= '5.9.0') {
+    if (require('semver').gte(cds.version, '5.9.0')) {
```
Comment (chgeo) [inline test/consuming-services.test.js:35]: Please add `semver` to the devDpendencies of this project.
(it only works at the moment due to our dev setup, where `semver` is already available).

**Review by chgeo — CHANGES_REQUESTED** [review]

File: test/consuming-services.test.js (hunk 1 above)
Comment (MikhailGoncharov) [reply test/consuming-services.test.js:35]: done

File: package.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
     "chai": "^4.3.4",
     "chai-as-promised": "^7.1.1",
     "chai-subset": "^1.6.0",
+    "semver": "^7.3.5",
```
Comment (MikhailGoncharov) [inline package.json:23]:
```suggestion
    "semver": "^6.3.0",
```

File: test/consuming-services.test.js (hunk 1 above)
Comment (chgeo) [reply test/consuming-services.test.js:35]: Why not use latest `semver` 7 ?
I think you need to update the package-lock as well with `npm update`

Comment (MikhailGoncharov) [reply test/consuming-services.test.js:35]: Because it is the latest supported by node 12. I took 7 initially.

Comment (chgeo) [reply test/consuming-services.test.js:35]: I have bumped semver to 7 and updated package-lock

**Review by chgeo — APPROVED** [review]

**Pull request merged**